.github/workflows/build.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: '14'
+      - run: npm i -g npm@8
```

Comment:
we migrated to the new NPM package-lock format but stay on Node.js 14 thats why we need to manually install new NPM.